modules/@apostrophecms/doc-type/ui/apos/components/AposDocContextMenu.vue

```diff
@@ -443,10 +451,40 @@ export default {
     },
     close() {
       this.$emit('close', this.doc);
+    },
+    checkIfConditions(doc, conditions) {
```

Comment:
Can we factor this to an `import`-ed file so it is shared with the other implementation?

Reply:
We could be the other implementations are different, for example they don't support the `$ne` for now neither nested properties (because it doesn't exist for fields).
I will extract it but I won't refactoring now other implementations since they are not the same.